torch/csrc/jit/python_ir.cpp

```diff
@@ -541,6 +541,8 @@ void initPythonIRBindings(PyObject* module_) {
         return types;
       });
   py::class_<ListType, Type, std::shared_ptr<ListType>>(m, "ListType")
+      .def(
+          py::init([](TypePtr a) { return ListType::create(a); }))
```

Comment:
Did you mean to change the permissions on this file (I think that's what these numbers are?)

![image](https://user-images.githubusercontent.com/5652049/50782498-2fdd8b80-1276-11e9-9255-6ca667b4b60d.png)